batchControl_test.go

```diff
@@ -315,3 +315,24 @@ func BenchmarkBatchControlLength(b *testing.B) {
 		testBatchControlLength(b)
 	}
 }
+
+func TestBatchControl__SetValidation(t *testing.T) {
+	bh := NewBatchControl()
+	bh.SetValidation(&ValidateOpts{})
+
+	// nil out
+	bh = nil
+	bh.SetValidation(&ValidateOpts{})
+}
+
+func TestBatchControl__IsValidWithPreserveSpacesOpt(t *testing.T) {
```

Comment:
Wanted to ensure nothing relied on spaces being trimmed when validating

Reply:
Thanks